tests/unit/mocks/mock-request-test.js

```diff
@@ -13,6 +13,18 @@ module('MockRequest', function(hooks) {
   setupTest(hooks);
   inlineSetup(hooks, serializerType);
 
+  module('#withParams', function() {
+    test('works with arrays', async function(assert) {
```

Comment:
This already passes. I guess this is because `MockRequest` is mostly used only for `GET` requests. I'm guessing that it might break for `POST` requests (which would be unusual but could happen with custom adapters). It might be difficult to setup this type of scenario and I think there's still value in this PR without doing this.

Reply:
Adam, you could write a test for the mock any with POST request ( taking array params )  because I use mock any with post / put alot myself so that would be useful ( if they don't already exist of course .. which i don't think they do )

Reply:
Hi. I did add a test for mock any with `GET` / `POST` / `PATCH` and `PUT` [here](https://github.com/danielspaniel/ember-data-factory-guy/pull/395/files#diff-2da6df751e64bdd490e7704016192aa6R223).

The test this comment is on (`mock-request-test.js`) is a test for `mock-store-request`. I don't think there's a good way right now to mock a `POST` request `withParams` for this type of request.

Reply:
In fact, I think I'll delete this test altogether because it's not relevant.

Reply:
yes, your last round em up and test em all iterator GET, PUT, POST , PATCH  was good idea.
  Are you trying to test every type of param (  string , array , hash ) or just array and string.  If so is it worth doing hash type ( or did I already do that elsewhere? )